bnd.bnd

```diff
@@ -0,0 +1,32 @@
+Bundle-Activator: \
+    ${classes;IMPLEMENTS;org.osgi.framework.BundleActivator}
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.6))"
+
+Export-Package:\
+    org.apache.sling.junit.annotations;version=1.1.0,\
+    org.apache.sling.junit;version=1.3.0,\
+    org.apache.sling.junit.rules;version=1.0.8
+
+-exportcontents:\
+    junit.framework;version=${junit.version},\
+    org.junit;version=${junit.version},\
+    org.junit.matchers.*;version=${junit.version},\
+    org.junit.rules.*;version=${junit.version},\
+    org.junit.runner.*;version=${junit.version},\
+    org.junit.runners.*;version=${junit.version},\
+    org.hamcrest;version=${hamcrest.version},\
+    org.hamcrest.*;version=${hamcrest.version}
+
+# bnd-maven-plugin use bndtools 4.2.0.  Macros for accessing repositories (ie. 
```

Comment:
I prefer first downloading the dependencies to a dedicated folder with https://maven.apache.org/plugins/maven-dependency-plugin/copy-mojo.html and then reference it from there.

Reply:
maven-dependency-plugin:copy approach has been used in the latest changes. Will do another pull request.